### PR TITLE
Create project

### PR DESCRIPTION
**Pull request by mkj-is**
This kick-off contains:

- Expectation transactions
- Promises testing
- Mockups

What remains to be added:

- UI testing bot
- Screenshot helpers

### CODE REVIEW

**Review by samoilyk — APPROVED** [review]

**Review by IGPenguin — APPROVED** [review]
Nice, některé ty věci by se dali fajnove zužitkovat i v UI testech, super práce 💪

File: Tests/FTTestingKitTests/MockupTests.swift [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+import FTTestingKit
+
+final class MockupTests: XCTestCase {
+    func testArrayGeneration() {
+        let strings = String.random(count: 5)
+        XCTAssertEqual(strings.count, 5)
+    }
+
+    func testDateRandomness() {
+        XCTAssertNotEqual(Date.random(), Date.random())
```
Comment (IGPenguin) [inline Tests/FTTestingKitTests/MockupTests.swift:11]: ❤️

**Comment by mkj-is** [comment]
Closing and reopening to kick-off Bitrise build.

**Comment by bezoadam** [comment]
@mkj-is  nechces este trochu upravit README.md?

**Review by bezoadam — APPROVED** [review]

**Review by peuf0u — APPROVED** [review]

**Review by eRDe33 — APPROVED** [review]

**Review by tomasbabulak — APPROVED** [review]

**Comment by mkj-is** [comment]
I added issue with documentation #2.

**Pull request merged**